client-bagel-buddy/src/containers/student/teacherList/teacherDetail.css

```diff
@@ -71,4 +71,63 @@
     margin-top: 16px;
     margin-bottom: 40px;
     font-size: 16px;
+}
+
+.teacher-info {
+    display: flex;
+    margin-top: 60px;
+    margin-bottom: 35px;
+}
+
+.teacher-img {
+    width: 220px;
+    height: 220px;
+    border-radius: 220px;
+    flex: 0;
+}
+
+.star-chunk {
+    flex: 1;
+    padding-left: 45px;
+}
+
+.teacher-name {
+    color: var(--Text, #3F1E07);
+
+    font-size: 28px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
```

Comment:
We already specified the font size/style/weight for `<h1>` in index.css, so no need line 97-100 here